### PR TITLE
Prog/#433 rare shows first cut

### PR DESCRIPTION
**Pull request by gammaw**
Please review this historical PR :)
The separate commits should help, e.g., with reviewing the usual transition madness. 
My logic for the initial frontend schedule was that I kept the shows with the latest last episode (mimicking the frontend logic we agreed upon: display the show with the last submitted episode). Below you find the current schedule to doublecheck against. 

image

### CODE REVIEW

File: wp/wp-content/themes/lahma_maker/Showsschedule.php [hunk 1]
```diff
@@ -44,6 +42,7 @@
 <div class="Thursday day">
     <h3>Thursday / Csütörtök</h3>
     <ul>
+        <li>12:00–14:00 <strong><a href="../shows/dalmata-radio/">Dalmata Radio</a></strong>
```
Comment (pvj) [inline wp/wp-content/themes/lahma_maker/Showsschedule.php:45]: Will we update this manually from now?

**Review by pvj — APPROVED** [review]
Thank you. I didn't spot any mistakes

File: wp/wp-content/themes/lahma_maker/Showsschedule.php (hunk 1 above)
Comment (gammaw) [reply wp/wp-content/themes/lahma_maker/Showsschedule.php:45]: Yes, that was the plan.

**Review by tuz666 — APPROVED** [review]
I'm pretty sure that @pvj fully checked it!

**Pull request merged**